RSSReader/View/FeedViewController.m

```diff
@@ -0,0 +1,87 @@
+//
+//  FeedViewController.m
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 17.11.20.
+//
+
+#import "FeedViewController.h"
+#import "FeedCell.h"
+
+@interface FeedViewController ()
+
+@end
+
+@implementation FeedViewController
+
+- (void)viewDidLoad {
+    self.navigationController.navigationBar.hidden = YES;
+    [super viewDidLoad];
+    [self setupPresenter];
+    [self setupTableView];
+    [self setupConstraints];
+}
+
+- (void)setupPresenter {
+    NSMutableArray *feedItemArray = [[NSMutableArray alloc] init];
+    self.feedItemArray = feedItemArray;
+    [feedItemArray release];
+    FeedPresenter *presenter = [[FeedPresenter alloc] initWithArray:self.feedItemArray];
+    self.presenter = presenter;
+    [presenter release];
+    [self.presenter loadNewsWiithCompletion:^(BOOL success) {
+        if (success) {
+            [self.tableView reloadData];
+        }
+    }];
+}
+
+- (void)setupTableView {
+    UITableView *tableView = [[UITableView alloc] init];
```

Comment:
Just implement lazy loading call and add all this creation and configuration there